docs/user-manual/en/security.md

```diff
@@ -1087,22 +1087,75 @@ amqp-sasl-gssapi {
 };
 ```
 
+#### SCRAM-SHA SASL Mechanism
```

Comment:
This (and the new sub sections below) seem to be at the wrong level, suggesting they are part of the "### Kerberos Authentication" section, which clearly they arent.

Looking further up, it seems there is a LoginModules section about each of the modules, and then the Kerberos one sits separately with its own section. Then this [almost] does the same.

I think the section depths should be fixed and this should go above the Kerberos section as its likely more widely applicable.

Reply:
good catch, will sort

Reply:
the two new login modules need their own little section in the login modules section too